src/metabase/channel/impl/slack.clj

```diff
@@ -70,19 +69,26 @@
 
   Nested `blocks` lists containing text cards are passed through unmodified."
   [attachments]
-  (letfn [(f [a] (select-keys a [:title :title_link :fallback]))]
-    (reduce (fn [processed {:keys [rendered-info attachment-name channel-id] :as attachment-data}]
-              (conj processed (if (:blocks attachment-data)
-                                attachment-data
-                                (if (:render/text rendered-info)
-                                  (-> (f attachment-data)
-                                      (assoc :text (:render/text rendered-info)))
-                                  (let [image-bytes (channel.render/png-from-render-info rendered-info slack-width)
-                                        {:keys [url]} (slack/upload-file! image-bytes attachment-name channel-id)]
-                                    (-> (f attachment-data)
-                                        (assoc :image_url url)))))))
-            []
-            attachments)))
+  (reduce (fn [processed {:keys [title title_link attachment-name rendered-info] :as attachment-data}]
+            (conj processed (if (:blocks attachment-data)
+                              attachment-data
+                              (if (:render/text rendered-info)
+                                {:blocks [{:type "section"
+                                           :text {:type "mrkdwn"
+                                                  :text (format "<%s|%s>" title_link title)}}
```

Comment:
Escaping?

Reply:
The guidance for `>` it to use htmlentities: https://api.slack.com/reference/surfaces/formatting#escaping

I'm guessing this works fine for pipe as well:

```clojure
(defn escape-slack-link-text [text]
  (when text
    (-> text
        (str/replace #"&" "&amp;")
        (str/replace #"<" "&lt;")
        (str/replace #">" "&gt;")
        (str/replace #"\|" "&124;"))))
```

I suppose we'd want to strip any accidental formatting as well though, this seems pretty thorny...

Reply:
If we're OK with the formatting, a Button block seems the safest:

```json
{
  "type": "button",
  "text": {
    "type": "plain_text",
    "text": "Link Button"
  },
  "url": "https://api.slack.com/block-kit"
}
```

Reply:
I did the straightforward thing in the end 07a2bcc5